MetaMask/metamask-mobile
# feat: New Ui for permissions dapp screen

**Description**
Added UI change on the permissions sheet.

**Screenshots/Recordings**
https://recordit.co/Rst9APMnfD
<img src=""
![image](https://github.com/MetaMask/metamask-mobile/assets/46944231/27d63dc2-a096-4e2e-8be7-195696e618dc)

dark mode:
<img width="352" alt="image" src="https://github.com/MetaMask/metamask-mobile/assets/46944231/229c9030-8741-4c2a-9b72-48d141be33aa">

<img width="356" alt="image" src="https://github.com/MetaMask/metamask-mobile/assets/46944231/f0204e80-325c-4156-a934-39be896ab4bf">


**Test scenario**
* Connect to a dapp
* open the account sheet
* Press permission
* Should have a new UI for the description
**Issue**

Progresses #???

**Checklist**
